Extension/src/LanguageServer/languageConfig.ts

```diff
@@ -55,7 +55,7 @@ function getMLSplitAfterPattern(): string {
 
 function getMLPreviousLinePattern(insert: string): string | undefined {
     if (insert.startsWith("/*")) {
-        return `\\A(?=^(\\s*(\\/\\*\\*|\\*)).*)(?=(?!(\\s*\\*\\/)))`;
```

Comment:
This was added to fix an issue with false positives on multi-line arithmetic.  It seems it breaks for comments too?
```
int n = 3
    * 4
    * 5  // Enter here results in an asterisk being added
```

Reply:
Ah, yeah, I repro that bug. What was the reasoning behind adding the "\A"? That appears to be just break "everything", right?

Anyway, I'll cancel this PR since it sounds like the actual fix is more complicated.

Reply:
@bobbrow TypeScript appears to have the same bug. So, I'm thinking we keep that bug and file a VS Code bug? It at least fixes the particular case mentioned in the original bug repro, which might be more common (i.e. the bug repro no requires 2 lines starting with the "*" instead of one).

Reply:
I tend to agree.  I would rather have the comments working than not.  @elahehrashedi do you have any objections?

Reply:
To answer your earlier question about \A:
![image](https://user-images.githubusercontent.com/12818240/112677287-c500d680-8e26-11eb-9539-594bcc1f3acc.png)

Reply:
Where are you getting that from? The regex parsers I have tried do not have that `'\A'` listed as valid.

Reply:
My mistake. I forgot to set the Language mode on regex101.com.